ecs-logs-collector.sh

```diff
@@ -404,13 +405,19 @@ get_open_files() {
 
 get_common_logs() {
   try "collect common operating system logs"
+  mkdir -p "$info_system"
 
-  dstdir="${info_system}/var_log"
-  mkdir -p "$dstdir"
+  # On AL2023, system logs are stored in the systemd journal
+  if grep -q "Amazon Linux 2023" /etc/os-release; then
```

Comment:
Q. would we have to update this line later for `Amazon Linux 202x`? or should we use wildcard here?

Reply:
this is a good point. I changed to check if `journalctl is available`, then we collect logs. works on all of AL2, AL2022, CentOS and Ubuntu!